src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -3057,7 +3057,179 @@ class C { }
             var compilation = CreateCompilation(text);
             Assert.False(compilation.GetSpecialType(SpecialType.System_ArgIterator).IsManagedTypeNoUseSiteDiagnostics);
             Assert.False(compilation.GetSpecialType(SpecialType.System_RuntimeArgumentHandle).IsManagedTypeNoUseSiteDiagnostics);
-            Assert.False(compilation.GetSpecialType(SpecialType.System_TypedReference).IsManagedTypeNoUseSiteDiagnostics);
+        }
+
+        [Fact, WorkItem(65530, "https://github.com/dotnet/roslyn/issues/65530")]
+        public void IsManagedType_TypedReference()
+        {
+            var libSrc = @"
+public unsafe class C
+{
+    public static System.TypedReference* M(System.TypedReference* r)
```

Comment:
>System.TypedReference* r [](http://example.com/codeflow?start=43&length=24)

Are we testing similar scenario with a local?

Reply:
Added.